lib/components/position_component.dart

```diff
@@ -203,6 +182,10 @@ abstract class PositionComponent extends Component {
     canvas.restore();
   }
 
+  @mustCallSuper
```

Comment:
I think this then "inherits" must call super from the parent

Reply:
It should be like that, but it is currently not:
https://github.com/dart-lang/sdk/issues/35372

Reply:
On the other hand then we have to add that to everything, I think we'll just leave out the annotation and just let them solve that bug.

Reply:
oof. yeah I think we can keep our code clean and rely on the annotation as more as a doc until they fix that bug